test/support/utils.h

```diff
@@ -524,12 +524,12 @@ class Wrapper
   public:
     Wrapper()
     {
-        my_field = ::std::shared_ptr<T>(new T());
+        my_field = ::std::make_shared<T>();
```

Comment:
If we are doing some changes there let's use member-init-list for all constructors then.

Reply:
Fixed.